### PR TITLE
Change frameDuration to microseconds

### LINKED ISSUE #600: Should we "un-round-up" OpusEncoderConfig.frameDuration to 2.5ms for a ptime of 3?
`OpusEncoderConfig.frameDuration` is defined as a valid `ptime`, according to Section 6.1 of [[RFC7587]](https://w3c.github.io/webcodecs/opus_codec_registration.html#biblio-rfc7587):

"Possible values are 3, 5, 10, 20, 40, 60, or an arbitrary multiple of an Opus frame size rounded up to the next full integer value [...]"

Supported Opus frame sizes in ms are {2.5, 5, 10, 20, 40, 60}. The 2.5 ms Opus frame size gets rounded up to a ptime of 3.

If we configure an AudioEncoder for Opus with a frameDuration of 3, is the underlying implementation supposed to un-round-up the 3ms value down to 2.5ms? Same goes for a frameDuration/ptime of 8, un-rounded-up to 7.5ms.

The confusion arises if 4 EncodedVideoChunks with a frameDuration of 3 are emitted: once decoded, should they total 10ms or 12ms? This should be clarified before browsers ship implementations.

A solution could be to change `OpusEncoderConfig.frameDuration` to use microseconds and only accept valid Opus frame sizes, and add an `OpusEncoderConfig.ptime` which controls the [repacketizer](https://www.opus-codec.org/docs/html_api/group__repacketizer.html) under the hood.

If it understood across the industry that a ptime of 3 maps to 2.5ms (and 8 to 7.5ms, and so on), there should probably be an implementer's note in the spec.

### PR DESCRIPTION
**Pull request by tguilbert-google**
Fixes #600.

***
<a href="https://pr-preview.s3.amazonaws.com/tguilbert-google/webcodecs/pull/602.html" title="Last updated on Nov 1, 2022, 7:09 PM UTC (648c3be)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/602/34b6ed2...tguilbert-google:648c3be.html" title="Last updated on Nov 1, 2022, 7:09 PM UTC (648c3be)">Diff</a>

### CODE REVIEW

**Review by stepancar — APPROVED** [review]

File: opus_codec_registration.src.html [hunk 1]
```diff
@@ -131,14 +131,14 @@
 <xmp>
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
-  [EnforceRange] unsigned long frameDuration;
+  [EnforceRange] unsigned long frameDuration = 20000;
```
Comment (dalecurtis) [inline opus_codec_registration.src.html:134]: unsigned long long like existing `duration` fields.

**Review by dalecurtis — APPROVED** [review]
lgtm % type

File: opus_codec_registration.src.html (hunk 1 above)
Comment (tguilbert-google) [reply opus_codec_registration.src.html]: Done.

**Comment by tguilbert-google** [comment]
Apologies for the above commit history... I did something wrong trying to rebase and resolve my merge conflicts, but the files changed seem adequate.

**Pull request merged**